pilz_teleoperation/src/pilz_teleoperation/teleoperation_driver.py

```diff
@@ -36,16 +35,16 @@ def __init__(self, twist=None, *args, **kwargs):
                                twist.angular.z)
 
     def project_on_plane(self, projection_plane):
-        if projection_plane == SetTeleopSettingsRequest.USE_XZ_PLANE:
-            self.linear.y, self.linear.z = 0, self.linear.y
-        elif projection_plane == SetTeleopSettingsRequest.USE_YZ_PLANE:
-            self.linear.x, self.linear.y, self.linear.z = 0, self.linear.x, self.linear.y
+        first = self.linear.x
+        second = self.linear.y
+        self.linear.x, self.linear.y = 0, 0
+        setattr(self.linear, projection_plane[0], first)
+        setattr(self.linear, projection_plane[1], second)
 
     def change_rotation_axis(self, axis):
-        if axis == 1:
-            self.angular.x, self.angular.y = 0, self.angular.x
-        elif axis == 2:
-            self.angular.x, self.angular.z = 0, self.angular.x
+        value = self.angular.x
+        self.angular.x = 0
+        setattr(self.angular, axis, value)
```

Comment:
When another plane/axis is toggled the standard jogging by x,y,z or rx, ry, rz is also changed.
This can resolve in unexprected behaviour.

Any suggestion how we handle this?

Reply:
Will be addressed in another PR